src/components/data/kanban/kanban.tsx

```diff
@@ -608,6 +610,11 @@ export const KanbanButton: React.FC<KanbanButtonProps> = ({
     <ButtonLink
       align="start"
       className="mykn-kanban__button"
+      disabled={
+        typeof object.disabled === "undefined"
+          ? !href
+          : Boolean(object.disabled)
+      }
```

Comment:
I wonder: wouldn't it be simpler to have a prop "disabled" and let the parent component manage the logic of whether the button is disabled?

Reply:
There are a few scenario's where an item's disabled can be set:

- There is a prop (object attribute) set that indicates disabled (`typeof object.disabled !== "undefined"`), in such case the prop should be used.
- There is no prop (object attribute) set that indicates disabled (`typeof object.disabled === "undefined"`), in such cased the kanban should base it's state on whether a `href` or `onClick` is set that provided interaction. 

I don't think the first case is used anywhere, but I can imagine that the automatic behavior (second scenario) might be problematic is certain cases so I think is sensible to provide an escape hatch.